### DISCUSSION TITLE
Which python version is currently supported for the nnom package?

### DISCUSSION THREAD

**Discussion by achenbachsven** [majianjia/nnom#173] [Q&A]
First of all, I would like to thank you Majianjia for your great work regarding this nnom package. You outperform many commercially available nn embedded software libs. I have integrated your lib in many projects. Actually, I use your library with python 3.6. Would it be possible to use python 3.9/3.10 too?

Best wishes
Sven

**Comment by majianjia** [comment]
havent try 3.9 but 3.8 should be fine.
